### ISSUE TITLE
lazyjson - OSError: Bad file descriptor

### ISSUE THREAD

**Issue by laerus** [xonsh/xonsh#1426]
this issue is bothering me for too long, a bug that appears 1 out of 5 times i startup xonsh:

```
$ XONSH_DEBUG=1 python3 -m xonsh
Exception ignored in: <bound method LazyJSON.__del__ of <xonsh.lazyjson.LazyJSON object at 0xb4a89fec>>
Traceback (most recent call last):
  File "/home/laerus/.local/lib/python3.4/site-packages/xonsh/lazyjson.py", line 212, in __del__
    self.close()
  File "/home/laerus/.local/lib/python3.4/site-packages/xonsh/lazyjson.py", line 217, in close
    self._f.close()
OSError: [Errno 9] Bad file descriptor
```

anyone else getting this? any ideas on debugging it?

**Comment by scopatz** [comment]
I don't see it, but I agree it is annoying.

**Comment by scopatz** [comment]
Potential fix in #1427
